packages/salesforcedx-utils-vscode/src/cli/commandExecutor.ts

```diff
@@ -73,11 +74,20 @@ export class CommandExecution {
         timerSubscriber.unsubscribe();
       }
     });
+    this.processCloseSubject = Observable.fromEvent(
```

Comment:
We probably don't need this. The issue seems to be a bit more subtle than just using Close vs Exit. See #111

Reply:
I had moved away from using the Executor as much as possible towards REST although there is more to do on describeGlobal.     I just updated Winston's last code to use Close instead of Exit as we had discussed...